### ISSUE TITLE
NCBI assembly summary now has 38 columns

### ISSUE THREAD

**Issue by schorlton** [pirovc/genome_updater#86]
Seems like NCBI updated their assembly summary format. Further details: https://ftp.ncbi.nlm.nih.gov/genomes/README_assembly_summary.txt

Getting an error failing to download, but upon investigation, it is downloading correctly and `check_assembly_summary()` is failing because the assembly summary has more than 23 columns. Steps to reproduce:

```bash
wget -O - ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/assembly_summary_refseq.txt | tail +3 | awk 'BEGIN{FS=OFS="\t"}{print NF}' | sort -u
```

**Comment by pirovc** [comment]
Indeed, thanks for the heads up, will release a fix today.

**Comment by pirovc** [comment]
Fixed with v0.6.3 #87
